apps/server/src/utils/coerceType.ts

```diff
@@ -40,3 +58,35 @@ export function coerceNumber(value: unknown): number {
   }
   return parsedValue;
 }
+
+/**
+ * @description Converts a value to a colour if possible, throws otherwise
+ * @param {unknown} value - Value to be converted to a colour.
+ * @returns {string} - The converted value as a string.
+ * @throws {Error} Throws an error if the value is null or undefined.
+ */
+const hexColor = new RegExp(/^#((?:[a-f\d]{1,2}){3,4})$/, 'i');
+export function coerceColour(value: unknown): string {
+  if (value == null || typeof value != 'string') {
+    throw new Error('Invalid colour value received');
+  }
+  if (value[0] == '#') {
+    const isHexColor = hexColor.exec(value);
+    if (isHexColor) {
+      //TODO: if the hex value is 4 or 8 digits the last 1 or 2 are the trasparent part
+      // we sould remove that to not breake the css formating the app
+      return value;
```

Comment:
why would we remove it?
I am not sure we can make these kinds of decisions on behalf of the user

We know sometimes it will break, but not always, should the user not be in control of this?

Reply:
Yes sound correct.
but we should then maybe look into having the `getAccessibleColour` add the transparent  part to its calculation